.github/workflows/workflow.yml

```
@@ -323,32 +310,30 @@ jobs:
         run: "${GITHUB_WORKSPACE}/.github/publish-git-tag.sh"
 
   publish-to-conda:
-    runs-on: "ubuntu-20.04"
+    runs-on: windows-latest
     needs: [ deploy, test-on-windows ]
     steps:
-      - uses: conda-incubator/setup-miniconda@v3
+      - uses: actions/checkout@v4
+      - name: Restore build
+        uses: actions/cache@v4
         with:
-          auto-update-conda: true
-          python-version: "3.9.9"
-          # Add conda-forge for OpenFisca-Core
-          channels: conda-forge
-          activate-environment: true
-      - name: Get source code
-        uses: actions/checkout@v4
-      - name: Get version
-        run: echo "PACKAGE_VERSION=$(python3 ./pyproject.toml --version)" >> $GITHUB_ENV
-      # Get the last commit hash on the PR (-2 : before the merge commit)
-      - uses: actions/github-script@v6
-        id: last_pr_commit
+          path: ${{ runner.temp }}\conda-bld
+          key: build-conda-${{ hashFiles('pyproject.toml') }}-${{ github.sha }}
+          fail-on-cache-miss: true
+      - uses: conda-incubator/setup-miniconda@v3
         with:
-          script: |
-            const commits = ${{ toJSON(github.event.commits) }}
-            return commits.at(-2).id;
-          result-encoding: string
-      - name: Conda build and upload
-        # This shell is made necessary by https://github.com/conda-incubator/setup-miniconda/issues/128
+          activate-environment: openfisca
+          miniforge-version: latest
+          python-version: 3.9.12
```

Aligner les versions de python ?
```suggestion
          python-version: 3.10.6
```